### ISSUE TITLE
Could not find any matches for org.linphone:linphone-sdk-android:5.4.+

### ISSUE THREAD

**Issue by Cersin** [BelledonneCommunications/linphone-android#2291]
Can't build

9: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':app:mergeDebugNativeLibs'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Could not find any matches for org.linphone:linphone-sdk-android:5.4.+ as no versions of org.linphone:linphone-sdk-android are available.
     Required by:
         project :app

**Comment by Viish** [comment]
Hi,

Sorry about that, we are migrating our website and it broke the redirection.
We'll fix it asap, in the meantime you can change the repo URL to https://download.linphone.org/maven_repository/

Cheers,

**Comment by dendrocyte** [comment]
I can not reach linphone, even I change the repo's url. Any help?

**Comment by mindAndroid** [comment]
HI, i have added  maven { url "https://storage.googleapis.com/r8-releases/raw" } on build.gradle but still it does not find error said 
Could not find any matches for org.linphone:linphone-sdk-android:5.0+ as no versions of org.linphone:linphone-sdk-android are available.

**Comment by soriyasun** [comment]
HI, all
Has this issue been fixed ? I've exactly the same problem. 
Best regards

**Comment by Cersin** [comment]
> HI, all Has this issue been fixed ? I've exactly the same problem. Best regards

rebase your fork repo

**Comment by mindAndroid** [comment]
> HI, all Has this issue been fixed ? I've exactly the same problem. Best regards

yes its fix for me after i update the meven url to "https://download.linphone.org/maven_repository/"

**Comment by Viish** [comment]
Problem is now fixed.
